tests/test_statistics.py

```diff
@@ -1,37 +1,37 @@
 import numpy
 import pytest
+from numpy.testing import assert_allclose
 
 import dpnp
 
 from .helper import get_all_dtypes
 
 
 @pytest.mark.parametrize(
-    "type",
-    [numpy.float64, numpy.float32, numpy.int64, numpy.int32],
-    ids=["float64", "float32", "int64", "int32"],
+    "dtype", get_all_dtypes(no_none=True, no_bool=True, no_complex=True)
 )
 @pytest.mark.parametrize("size", [2, 4, 8, 16, 3, 9, 27, 81])
-def test_median(type, size):
-    a = numpy.arange(size, dtype=type)
+def test_median(dtype, size):
+    a = numpy.arange(size, dtype=dtype)
     ia = dpnp.array(a)
 
     np_res = numpy.median(a)
     dpnp_res = dpnp.median(ia)
 
-    numpy.testing.assert_allclose(dpnp_res, np_res)
+    assert_allclose(dpnp_res, np_res)
 
 
 @pytest.mark.usefixtures("allow_fall_back_on_numpy")
 @pytest.mark.parametrize("axis", [0, 1, -1, 2, -2, (1, 2), (0, -2)])
 def test_max(axis):
-    a = numpy.arange(768, dtype=numpy.float64).reshape((4, 4, 6, 8))
+    dtype = dpnp.default_float_type()
```

Comment:
can we extend the test to all supported dtypes, like `get_all_dtypes(no_none=True, no_bool=True, no_complex=True)`